src/main/java/org/openjdk/jextract/JextractTool.java

```diff
@@ -383,14 +360,8 @@ private int run(String[] args) {
         parser.accepts("--output", "help.output", true);
         parser.accepts("-t", List.of("--target-package"), "help.t", true);
         parser.accepts("--version", "help.version", false);
-
-        if (isMacOSX) {
-            parser.accepts("--mac-framework-dir", "help.mac.framework", true);
-            parser.accepts("-f", "help.framework.library.path", true);
-        }
-
-        parser.addPlatformOption("--mac-framework-dir");
-        parser.addPlatformOption("-f");
+        parser.accepts("--macos-framework", "help.mac.framework", true);
+        parser.accepts("-f", "help.framework.library.path", true);
```

Comment:
In clang these are named `-F` and `-framework` [1] (there's no `-f`). I think we should just match that. We did the same for `-D`, `-I` and `-l` to an extent.

[1]: https://clang.llvm.org/docs/ClangCommandLineReference.html

Reply:
I mean -- if one has a prefix `macos-` the other should too -- or none should have it.

Reply:
Also, `-f` usually is used in conjunction with other options e.g. `-fmove-loop-invariants`. Which is probably the reason behind the capital letter `-F`.

Reply:
I'm fine with that